src/routes/landing/utils/Section.svelte

```diff
@@ -2,8 +2,8 @@
   export let tinted: boolean = false;
 </script>
 
-<section class="{tinted ? 'bg-gray-50 dark:bg-gray-800' : ''} mx-auto max-w-8xl">
-  <div class="mx-auto px-4 lg:px-20 py-8 {$$props.class ?? ''}">
+<section class={tinted ? 'bg-gray-50 dark:bg-gray-800' : ''}>
+  <div class="mx-auto max-w-8xl px-4 lg:px-20 py-8 {$$props.class ?? ''}">
```

Comment:
How can I remove / override this div class? In my case its unnecessarily adding padding on both sides and unable to take the full width available.

![image](https://github.com/user-attachments/assets/8873bb63-3f0f-4e7a-a28f-d1df8c123bed)